### DISCUSSION TITLE
VerneMQ Release 1.12.5

### DISCUSSION THREAD

**Discussion by ioolkos** [vernemq/vernemq#1966] [General]
Changelog:

- Change `git` clone protocol to `https` globally. 
- Update erlang-pbkdf2 and mongodb-erlang, fixes Issue https://github.com/vernemq/vernemq/issues/1928.
- Fix `proxy_protocol_use_cn_as_username` config for WS listeners: the CN now overrides the MQTT username correctly.
- Add `?P_RESPONSE_INFO` property in CONNACK when `auth_on_register_m5_hook` sets the property.
- Add per-listener `allow_anonymous_override` config (to open a single listener while allow_anonymous=off).
- Update `certifi` to 2.9.0.
- Update `epgsql` to 4.6.0.
- Update `jsx` to 3.1.0 (vmq_diversity, vmq_webhooks).
- Update `eredis` to 1.4.1 (vmq_diversity).

Note: the Binary VerneMQ packages (EULA-based) will be available later in a repository accessible to Github sponsors.

<hr /><em>This discussion was created from the release <a href='https://github.com/vernemq/vernemq/releases/tag/1.12.5'>VerneMQ Release 1.12.5</a>.</em>

**Comment by baoquach14** [comment]
Hi All

From https://github.com/vernemq/vernemq/issues/983 it mentioned VerneMQ  version 2.0 likely to support Windows.
Can you please confirm and when VerneMQ  version 2.0 will be release ? 
We customer around the still using Windows Server, we like to VerneMQ cluster on 3 Windows Servers part of product release around May 2023.    

Thank you for your help and support

Regards Bao

**Comment by agronholm** [comment]
> Note: the Binary VerneMQ packages (EULA-based) will be available later in a repository accessible to Github sponsors.

Does this mean that the official Docker images won't ever be updated again? Or that binary packages won't be public?
